packages/editor-ui/index.html

```diff
@@ -8,6 +8,7 @@
 		<script type="text/javascript">
 			window.BASE_PATH = '/{{BASE_PATH}}/';
 			window.REST_ENDPOINT = '{{REST_ENDPOINT}}';
+			window.n8nHooksNext = true;
```

Comment:
we should just remove this line. The external hooks simply won't be loaded anymore https://github.com/n8n-io/n8n-hosted-backend/pull/816